modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/type/deployer/CETDeployerImpl.java

```diff
@@ -200,6 +209,16 @@ private List<ServiceRegistration<?>> _deploy(
 					jsImportMapsEntryCET.getURL())));
 	}
 
+	private List<ServiceRegistration<?>> _deploy(ThemeCSSCET themeCSSCET) {
+		return Arrays.asList(
+			_bundleContext.registerService(
```

Comment:
The current _register method receives an objects that extends `Registrable` (e.g: [IFrameCETPortlet](https://github.com/liferay/liferay-portal/blob/master/modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/portlet/IFrameCETPortlet.java)). That kind of class holds some logic storing fields, rendering stuff, etc. So since in our case we just need to pass a single parameter (`external.reference.code`), we judge creating a class that extends `Registrable` unnecessary.